src/python/pants/backend/python/lint/isort/subsystem.py

```diff
@@ -2,9 +2,18 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
Note the file changed locations. V1 still works as you'd expect despite this change.